_incObj/sub CalcSqrt.asm

```diff
@@ -0,0 +1,45 @@
+; ---------------------------------------------------------------------------
+; Subroutine calculate a square root (only available in REV00)
+
+; input:
+;	d0 = number
+
+; output:
+;	d0 = square root of number
+; ---------------------------------------------------------------------------
+
+; ||||||||||||||| S U B	R O U T	I N E |||||||||||||||||||||||||||||||||||||||
+
+
+		if Revision=0
+CalcSqrt:
+		movem.l	d1-d2,-(sp)
+		move.w	d0,d1
+		swap	d1
+		moveq	#0,d0
+		move.w	d0,d1
+		moveq	#7,d2
+
+	loc_2C80:
```

Comment:
Perhaps rename loc_2C80 to loop and loc_2C9A to jump, both prefixed with at symbols?